modules/cpmso-ts-mhc-etcd-degraded.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="cpmso-ts-mhc-etcd-degraded_{context}"]
+= Recovering a degraded etcd Operator after a machine health check operation
+
+While performing remediation, the machine health check might delete a control plane machine that is hosting etcd. If the etcd member is not reachable at that time, the etcd Operator becomes degraded. Manual intervention is required to force the etcd Operator to remove the failed member and restore the cluster state.
+
+.Procedure
+
+. List the control plane machines in your cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get machines -l machine.openshift.io/cluster-api-machine-role==master -n openshift-machine-api -o wide
+----
++
+A failed control plane machine has the `STATE` value `stopped`.
```

Comment:
An educated guess on my part :sweat_smile:

Reply:
Hmm, this may be true in some cases, another hint would be that the Machine is stuck in the `Deleting` phase for an extended period.

If the etcd operator is degraded and the Machine is phase Failed or Deleting, that would be an indicator

Reply:
Ok -I pulled `Stopped` from the `oc get machines` output in the etcd troubleshooting examples. So is it that case then that any of the following would tip the user off?

1. a `STATE` value of `stopped`
2. a `PHASE` value of `Failed`
3. a `PHASE` value of `Deleting` that doesn't resolve for `<some number of>` minutes